src/components/AssetManagementPanel/AssetManagementApplication/index.tsx

```diff
@@ -37,12 +41,17 @@ export const AssetManagementApplication = ({
   const onSetFormAction = (AssetManagementActions: AssetManagementActions) => {
     setAssetManagementAction(AssetManagementActions);
     setHolderTransferringState("");
+    setBeneficiaryEndorseState("");
   };
 
   useEffect(() => {
     setHolderTransferringState(changeHolderState);
   }, [changeHolderState]);
 
+  useEffect(() => {
```

Comment:
why?????

Reply:
ok i removed the extra state for the error state.